### ISSUE TITLE
WebPack Warnings

### ISSUE THREAD

**Issue by kb3eua** [akserg/ng2-slim-loading-bar#12]
I'm getting warnings using this with WebPack.  Seems to be complaining that no *.ts files exist in the package:

```
WARNING in ./~/ng2-slim-loading-bar/index.js
Cannot find source file 'index.ts': Error: Cannot resolve 'file' or 'directory' ./index.ts in C:\redacted\node_modules\ng2-slim-loading-bar
WARNING in ./~/ng2-slim-loading-bar/src/slim-loading-bar.component.js
Cannot find source file 'slim-loading-bar.component.ts': Error: Cannot resolve 'file' or 'directory' ./slim-loading-bar.component.ts in C:\redacted\node_modules\ng2-slim-loading-bar\src
WARNING in ./~/ng2-slim-loading-bar/src/slim-loading-bar.service.js
Cannot find source file 'slim-loading-bar.service.ts': Error: Cannot resolve 'file' or 'directory' ./slim-loading-bar.service.ts in C:\redacted\node_modules\ng2-slim-loading-bar\src
WARNING in ./~/ng2-slim-loading-bar/src/slim-loading-bar.utils.js
Cannot find source file 'slim-loading-bar.utils.ts': Error: Cannot resolve 'file' or 'directory' ./slim-loading-bar.utils.ts in C:\redacted\node_modules\ng2-slim-loading-bar\src
Child html-webpack-plugin for "index.html":
```

**Comment by akserg** [comment]
Thanks.
I will have a look.

**Comment by kb3eua** [comment]
@akserg I was able to find a workaround by excluding it from the source-map loader.  It looks like the ".map" files in the package have references to the *.ts source files, which of course can't be found.
```

preLoaders: [
    {
        test: /\.js$/,
        loader: 'source-map',
        exclude: [
            // these packages have problems with their sourcemaps
            root('node_moduels/rxjs'),
            root('node_modules/@angular'),
            root('node_modules/ng2-slim-loading-bar')
        ]
    }
]
```

**Comment by akserg** [comment]
Hi @kb3eua 
Thank you for help. Fix available in 1.4.1
